rskj-core/src/main/java/co/rsk/peg/feeperkb/FeePerKbSupport.java

```diff
@@ -0,0 +1,105 @@
+package co.rsk.peg.feeperkb;
+
+import co.rsk.peg.abi.*;
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.peg.AddressBasedAuthorizer;
+import co.rsk.peg.BridgeSerializationUtils;
+import co.rsk.peg.feeperkb.constants.*;
+import org.ethereum.core.SignatureCache;
+import org.ethereum.core.Transaction;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+public class FeePerKbSupport {
+    private static final Integer FEE_PER_KB_GENERIC_ERROR_CODE = FeePerKbResponseCodes.getGenericErrorCode();
+    private static final Integer NEGATIVE_FEE_PER_KB_ERROR_CODE = FeePerKbResponseCodes.getNegativeFeeErrorCode();
+    private static final Integer EXCESSIVE_FEE_PER_KB_ERROR_CODE = FeePerKbResponseCodes.getNegativeFeeErrorCode();
+
+    private final FeePerKbStorageProvider provider;
+    private final FeePerKbConstants feePerKbConstants;
+    private static final Logger logger = LoggerFactory.getLogger("FeePerKbSupport");
```

Comment:
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(FeePerKbSupport.class);
```
It's better to use the class name, so if it changes it also changes in the log